recipes/views.py

```diff
@@ -117,6 +117,7 @@ def recipe_create(request):
             recipe.created_by = request.user
             recipe.modified_by = request.user
             recipe.save()
+            form.save_m2m()
```

Comment:
wow ok, was this an uncaught bug?

Reply:
ah good catch, yeah commit=False is tricky in that now you're responsible, good debugging!